xml/admin_software.xml

```diff
@@ -279,6 +279,15 @@
 
   <sect2 xml:id="sec.admin.software.transactional-updates.installation">
    <title>Applying Updates</title>
+   <note>
+    <title>Registering nodes</title>
+    <para>
+     In order to update nodes, you should first register them at SUSE Customer 
```

Comment:
It should say "you must first register..." because without registration it simply won't work. "Should" implies that it's possible.